react/src/MultiSelect/MultiSelect.stories.tsx

```diff
@@ -103,6 +103,24 @@ DisabledWithSelection.args = {
   values: ['What happens when the label is fairly long', 'Bat'],
 }
 
+export const WithFixedItemHeight = Template.bind({})
+WithFixedItemHeight.args = {
+  size: 'md',
+  fixedItemHeight: 68,
```

Comment:
we should add `defaultIsOpen: true,` to args so the story renders properly with the options shown

Reply:
I do for you

Reply:
thanks !